WordPress/Classes/Services/WordPressComSyncService.swift

```diff
@@ -28,23 +45,21 @@ class WordPressComSyncService {
         }
 
         let onSuccessInternal = {
```

Comment:
I think we can now get rid of the whole property here, it reads a bit confusing to me if it's only just that one line.

Reply:
Don't we need the block to wrap the account?